src/components/Nav.js

```diff
@@ -1,5 +1,79 @@
 import React from 'react';
+import styled from 'styled-components';
+import { BiSearch } from 'react-icons/bi';
 
 export default function Nav() {
-  return <div>This is Nav</div>;
+  const buttonList = [
+    { id: 1, name: '고객센터' },
+    { id: 2, name: '관심상품' },
+    { id: 3, name: '마이페이지' },
+    { id: 4, name: '로그인' },
+  ];
+
+  return (
+    <NavWrap>
+      <NavTop>
+        {buttonList.map(el => {
+          return <Span key={el.id}>{el.name}</Span>;
```

Comment:
```suggestion
          return <Menu key={el.id}>{el.name}</Span>;
```
span 태그는 기능적인 이름이고 이 스타일드 컴포넌트의 역할이 좀 더 드러나는 네이밍이 좋을거 같아요

Reply:
위의 답변사항과 마찬가지로 `map` 메소드는 삭제했습니다.
컴포넌트명 또한 역할이 잘 드러나게끔 지을 수 있도록 고민하겠습니다.